source/fundamentals/enterprise-auth.txt

```diff
@@ -0,0 +1,216 @@
+====================================
+Enterprise Authentication Mechanisms
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In this guide, we show you how to authenticate with MongoDB using each
+**authentication mechanism** available exclusively in the MongoDB Enterprise
+Edition. Authentication mechanisms are processes by which the driver and
+server confirm identity and establish trust to ensure security.
+
+The mechanisms that you can use with the latest version of the MongoDB
+Enterprise Edition are as follows:
+
+- :ref:`Kerberos (GSSAPI/SSPI) <gssapi-auth-mechanism>`
+- :ref:`LDAP (PLAIN) <plain-auth-mechanism>`
+
+If you need to authenticate using another mechanism, see the
+:doc:`Authentication Mechanisms guide </fundamentals/auth>`. For more
+information on establishing a connection to your MongoDB cluster, read our
+:doc:`Connection Guide </fundamentals/connection>`.
+
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+.. include:: /includes/fundamentals/auth-specify.rst
+
+Mechanisms
+----------
+
+.. _gssapi-auth-mechanism:
+
+Kerberos (GSSAPI/SSPI)
+~~~~~~~~~~~~~~~~~~~~~~
+
+*Available in MongoDB Enterprise Edition 3.7 and later.*
+
+The Generic Security Services API (``GSSAPI``) authentication mechanism
+allows the user to authenticate to a Kerberos service using the user's
+principal name.
+
+.. note::
+   The method refers to the ``GSSAPI`` authentication mechanism instead
+   of ``Kerberos`` because the driver authenticates using the
+   `GSSAPI RFC-4652 <https://tools.ietf.org/html/rfc4752>`_  SASL
+   mechanism.
+
+The following code snippets show how to specify the authentication mechanism,
+using the following placeholders:
+
+* ``username`` - your URL-encoded principal name, e.g. "username%40REALM.ME"
+* ``hostname`` - network address of your MongoDB server, accessible by your client
+* ``port`` - port number of your MongoDB server
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions and sample code for specifying this authentication
+mechanism:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify the GSSAPI authentication mechanism using a connection
```

Comment:
I agree that listing them is easier to read. Let me try using a list instead of a table. This probably also applies to the non-enterprise mechanisms, but might be beyond the scope of the PR.